gdal/data/nitf_spec.xsd

```diff
@@ -130,6 +130,8 @@
                     <xs:enumeration value="string"/>
                     <xs:enumeration value="integer"/>
                     <xs:enumeration value="real"/>
+                    <xs:enumeration value="float_raw"/>
```

Comment:
Was wondering if "IEEE754Float32BigEndian" (or something shorter) would be more descriptive ? But as the very least put that as a XML comment

Reply:
I went for the long name since the description is built into a functional element and thus easier to enforce than a comment.